src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Square extends Figure {
```

Comment:
The class implementation should not begin with an empty line. Please remove the empty line after the class declaration.